packages/eslint-config-deloitte-react/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "This package provides Deloitte Digital's code standards as an ESLint extensible config",
   "main": "./index.js",
   "scripts": {
-    "lint": "eslint --config node.js \"**/*.js\"",
+    "lint": "./node_modules/.bin/eslint --config node.js \"**/*.js\"",
```

Comment:
Did you find it was necessary to change `eslint` to `./node_modules/.bin/eslint`?